cocos/core/data/utils/preprocess-class.ts

```diff
@@ -92,7 +92,7 @@ function parseType (val, type, className, propName): void {
         if (type.length > 0) {
             val.type = type = type[0];
         } else {
-            return errorID(5508, className, propName);
+            errorID(5508, className, propName);
```

Comment:
Should keep `return` logic;

Reply:
The return type is void.

Reply:
I know, but we should not execute the code below.